api/src/main/java/ai/djl/ndarray/NDArray.java

```diff
@@ -3644,6 +3644,13 @@ default NDArray argSort(int axis) {
      */
     NDArray isInfinite();
 
+    /**
+     * Inverses the {@code NDArray}.
+     *
+     * @return inverse {@code NDArray}.
+     */
```

Comment:
Should we specify that this will return the matrix inverse of a _square_ matrix if it exists?

Reply:
Sure, Will specify it.